src/olympia/reviewers/serializers.py

```diff
@@ -386,10 +386,32 @@ def __init__(self, *args, **kwargs):
         # on `instance` being set correctly.
         self.fields['version'].output.instance = self.context['version']
 
+    def get_or_default(self, key, data, default=''):
+        """Return the value of ``key`` in ``data``
+
+        If that key is not present then return the value of ``key`` from
+        ``self.instance`, otherwise return the ``default``.
+
+        This method is a helper to simplify validation for partial updates.
+        """
+        return data.get(key, getattr(self.instance, key, default))
```

Comment:
Is it common to be requesting a property on an instance that doesn't exist?  (That seems more likely to be due to an error in the code?)
When I read the description I assumed it would be `data.get(key, getattr(self.instance, key) or default)`

Reply:
Indeed, I agree. I think it'd even be better to raise an error if an attribute doesn't exist, I'll change the code.

I guess it's just been bad habit.